app/src/main/java/fr/paug/androidmakers/model/ScheduleSlot.kt

```diff
@@ -0,0 +1,12 @@
+package fr.paug.androidmakers.model
+
+class ScheduleSlot(val room: Int, val sessionId: Int, val startDate: Long, val endDate: Long) {
```

Comment:
Would it be better to go for a data class? :)

Reply:
I always try to avoid data classes if I can because of https://jakewharton.com/public-api-challenges-in-kotlin/

Also me: [converts to a data class in the next commit](https://github.com/paug/AndroidMakersApp/commit/6b1f38c2a00a71532d209bf59e81b898f8a13178#diff-f526a99ff814dddb89d46d5e2219e2bf68ae4458e07cdbb09c14209570d3a661R3) 🤷

Reply:
Ah did not know about that article I use data classes everywhere 😅 🙈

Reply:
It's not part of a public API (yet?) so it's mostly ok I think but there's definitely a hidden cost to data classes. We'll see later :P 😇